### DISCUSSION TITLE
Do I need to logarithmize (log1p) & scale the 'corrected' counts after sctransform normalization?

### DISCUSSION THREAD

**Discussion by debojyoti21** [satijalab/seurat#6051] [Q&A]
Hi! I am working on single-cell RNA sequencing of bacteria. To perform normalization on my scRNAseq data, I am using the sctransform method and package developed by your group.

The sctransform Seurat [vignette](https://satijalab.org/seurat/articles/sctransform_vignette.html) recommends using the dense matrix of Pearson residuals directly as input for PCA and downstream clustering operations using algorithms like the Louvain algorithm. However, for my small bacterial single-cell RNA sequencing dataset of around 1000 cells, I also want to use the sctransform-normalized data to perform firstly, sc3s consensus clustering instead of the neighborhood graph-based Leiden clustering, and secondly, use network inference tools like Inferelator to determine gene regulatory networks.

Both these methods, the sc3s clustering, and the Inferelator, recommend using the sparse normalized count matrix as input instead of the dense Pearson residual matrix (I confirmed this from the groups that developed the respective methods). So, instead of the Pearson residuals, I want to use the 'corrected' UMI counts that sctransform computes and stores in projectname[["SCT"]]@count. In this regard, since the Pearson residual matrix requires neither further logarithmizing nor scaling, I am not sure if I need to (1) logarithmize and (2) scale the 'corrected' counts matrix before feeding the 'corrected' count matrix as input for downstream analysis like PCA. Please clarify.

**Comment by saketkc** [comment]
Answered on email:

You can get corrected counts from a sctrasnform object using: `corrected_counts <- GetAssayData(object = object, assay = "SCT", slot = "counts")`. This is a sparse matrix where the counts have been adjusted for difference in sequencing depth. A log1p() transform is useful generally to minimize the impact of outliers. Choice of scaling + standardizing is very context dependent.
